contracts/queue/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "queue"
+version = "0.7.2"
+authors = ["Ethan Frey <ethanfrey@users.noreply.github.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[profile.release]
+opt-level = 3
+debug = false
+rpath = false
+lto = true
+debug-assertions = false
+codegen-units = 1
+panic = 'abort'
+incremental = false
+overflow-checks = true
+
+[features]
+default = ["cranelift"]
+# For quicker tests, cargo test --lib. for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces", "cosmwasm-vm/backtraces"]
+cranelift = ["cosmwasm-vm/default-cranelift"]
+singlepass = ["cosmwasm-vm/default-singlepass"]
+
+[dependencies]
+cosmwasm-std = { path = "../../packages/std" }
+schemars = "0.6"
+serde = { version = "1.0.103", default-features = false, features = ["derive"] }
+snafu = { version = "0.5.0", default-features = false, features = ["rust_1_30"] }
+
+[dev-dependencies]
+cosmwasm-vm = { path = "../../packages/vm", default-features = false }
+serde_json = "1.0"
```

Comment:
Note to myself: this is not from cosmwasm-template. For the contracts in here we should add some kind of guard to make sure they cannot be published (like `private: true` in npm).

Reply:
```toml
[package]
publish = false
```

does this. will add

Reply:
Nice. Can you add this to both contracts?

Reply:
Yup, did that already